packages/polaris-viz/src/components/shared/HorizontalBars/HorizontalBars.tsx

```diff
@@ -78,6 +79,8 @@ export function HorizontalBars({
           return;
         }
 
+        const newAnimationDelay = animationDelay + SERIES_DELAY * seriesIndex;
```

Comment:
Just a suggestion: maybe we want to think of a better name for `newAnimationDelay` ?  🤔